feature_layers/display-subtype-feature-layer/README.md

```diff
@@ -0,0 +1,36 @@
+# Display a subtype feature layer
+
+Displays a composite layer of all the subtype values in a feature class.
+
+![](DisplaySubtypeFeatureLayer.png)
+
+## Use case
+
+This is useful for controlling labeling, visibility and symbology of a given subtype as though they are distinct layers on the map.
+
+## How to use the sample
+
+The sample loads with the sublayer visible on the map. Toggle its visibility by clicking the "Show sublayer" checkbox. To change the sublayer's renderer, click the "Toggle Renderer" button, and click the "Set Minimum Scale" button to set its minimum scale to that of the current map scale. Zoom in and out to see the sublayer become visible based on its new scale range.
+
+## How it works
+1.  Create a `SubtypeFeatureLayer` from a `ServiceFeatureTable` that defines a subtype, and add it to the `ArcGISMap`.
+2. Get a `SubtypeSublayer` from the subtype feature layer using its name.
+3. Enable the sublayer's labels and define them with `getLabelDefinitions()`.
```

Comment:
Should we be using `.` before method calls? i.e. `.getLabelDefinitions()`? Would apply to a few of them below as well.

Reply:
good question. I looked at some of our other readmes, and they don't use the `.` before the method. As I recall, we don't use it since it should be obvious to the reader which object is calling that method (e.g. in step 3 above it should be obvious that it's the SubtypeSublayer that the getLabelDefinitions() would be called on). If we're using the full object with method reference then that should be referred to as `SubtypeSublayer.getLabelDefinitions()`.